docs/src/main/asciidoc/project-features.adoc

```diff
@@ -37,4 +37,4 @@ If you want to learn more about any of the classes discussed in this section, yo
 
 If you are comfortable with {project-full-name}'s core features, you can continue on and read
 about
-<<advanced.html, {project-full-name}'s advanced features>>.
+<<documentation-overview.adoc#advanced-topics, {project-full-name}'s advanced features>>.
```

Comment:
I don't think that this is correct. What should change is that we link to the `advanced.html` file. @Buzzardo am I correct that you have to use the `link:` approach?

Also that's only `project-features` file. There was another reference in the `documentation-overvivew.adoc` that is wrong and needs to be fixed.

Reply:
Hey, @marcingrzejszczak! Do you mean I should link to the advanced.html file?

Reply:
Yes - https://docs.spring.io/spring-cloud-contract/docs/3.0.0/reference/html/advanced.html

Reply:
> I don't think that this is correct. What should change is that we link to the `advanced.html` file. @Buzzardo am I correct that you have to use the `link:` approach?
> 
> Also that's only `project-features` file. There was another reference in the `documentation-overview.adoc` that is wrong and needs to be fixed.

Can you please be more specific about the reference to fix in `documentation-overview.adoc` ?

Reply:
I've described it in the issue description https://github.com/spring-cloud/spring-cloud-contract/issues/1594

> https://docs.spring.io/spring-cloud-contract/docs/3.0.0/reference/html/documentation-overview.html#advanced-topics points to advanced.html.html

Reply:
> I've described it in the issue description #1594
> 
> > https://docs.spring.io/spring-cloud-contract/docs/3.0.0/reference/html/documentation-overview.html#advanced-topics points to advanced.html.html

Yes, thanks! I got it and I've tried to fix it. Please review the PR.